inc/dbutils.class.php

```diff
@@ -1820,4 +1820,42 @@ function getItemtypeForForeignKeyField($fkname) {
       $table = $this->getTableNameForForeignKeyField($fkname);
       return $this->getItemTypeForTable($table);
    }
+
```

Comment:
maybe better in CommonDBConnexity? ping @orthagh

Reply:
+1

Reply:
I've just realized  that call is done from Project, so maybe it can be moved into CommonDBTM, or it can stay here.

Reply:
I moved it to CommonDBTM and squashed my commits.